models/src_twilio.yml

```diff
@@ -437,7 +437,7 @@ sources:
             description: timestamp of when fivetran synced a record. 
 
           - name: created_at
-            description: 	The date that this resource was created, given in RFC 2822 format.
+            description: The date that this resource was created, given in RFC 2822 format.
```

Comment:
This was causing an error for me. This extra whitespace needed to be removed in order to run the package on my end.

Reply:
Extra whitespace removed